test/cpp/util/BUILD

```diff
@@ -104,5 +104,29 @@ cc_test(
     ],
 )
 
-
-
+cc_binary(
+    name = "grpc_cli",
+    srcs = [
+        "cli_call.cc",
+        "cli_call.h",
```

Comment:
As discussed on hangouts: `cc_binary()` has no `hdrs`, and even in `cc_library()` it's valid to list in `srcs` various `.h` files that are not intended to be consumed by other rules.

I am doing some work on reorganizing this anyway, for possible testability and reuse. I may send it as a new PR.